website/docs/r/admission_control_policy.html.markdown

```diff
@@ -0,0 +1,52 @@
+---
+layout: "dome9"
+page_title: "Check Point CloudGuard Dome9: dome9_admission_control_policy"
+sidebar_current: "docs-resource-dome9-admission-control-policy"
+description: |-
+  Creates admission control policies in Dome9
+---
+
+# dome9_admission_control_policy
+
+This  resource is used to  create and modify admission control policy in Dome9 for cloudguard containers. An admission control policy is the combination of a RuleSet applied to a specific Kubernetes cloud account with specific action.
```

Comment:
admission control policy in CloudGuard for ....

Reply:
done